mwaa-local-env

```diff
@@ -76,6 +77,9 @@ reset-db)
 start)
    docker-compose -f ./docker/docker-compose-local.yml up
    ;;
+start-celery)
+   docker compose -f ./docker/docker-compose-multi.yml up
```

Comment:
I think we should either stick with `celery` or `multi` as the nomenclature for this setup. Right now both names are used in various places.

Reply:
I do not have a preference here so I can change it to be multi as the nomenclature.